src/app/shared/dialog/dialog.component.html

```diff
@@ -1,12 +1,12 @@
-<form novalidate>
+<form novalidate aria-labelledby="modal-basic-title" role="dialog" aria-modal="true">
```

Comment:
Another case where I have concerns of the `id` being duplicated due to the possibility of multiple dialogs on a single page.